obs-browser-plugin.cpp

```diff
@@ -693,7 +694,7 @@ bool obs_module_load(void)
 
 #ifdef SHARED_TEXTURE_SUPPORT_ENABLED
 	obs_data_t *private_data = obs_get_private_data();
-	hwaccel = obs_data_get_bool(private_data, "BrowserHWAccel");
+	gpuCompositing = hwaccel = obs_data_get_bool(private_data, "BrowserHWAccel");
```

Comment:
This goes against the style guide we use as a reference.  See the Linux coding style guide linked from https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst#coding-guidelines:

> Don't put multiple statements on a single line unless you have something to hide:
> Don't put multiple assignments on a single line either. Kernel coding style is super simple. Avoid tricky expressions.

Reply:
Fixed